common/config/config.go

```diff
@@ -600,8 +600,8 @@ type (
 
 	// AsyncWorkflowQueueProvider contains the config for an async workflow queue. Only one field must be set.
 	AsyncWorkflowQueueProvider struct {
-		Type                     string    `yaml:"type"`
-		AsyncWorkflowQueueConfig *YamlNode `yaml:"config"`
+		Type   string    `yaml:"type"`
```

Comment:
please update the command above to explain supported `type`s and also corresponding config struct pointer.